muikku-timed-notifications-plugin/src/main/java/fi/otavanopisto/muikku/plugins/timed/notifications/NotificationController.java

```diff
@@ -17,12 +20,20 @@
 import fi.otavanopisto.muikku.plugins.communicator.model.CommunicatorMessage;
 import fi.otavanopisto.muikku.users.UserEmailEntityController;
 */
+import fi.otavanopisto.muikku.plugins.commonlog.LogProvider;
 
 @Dependent
 public class NotificationController {
   
   @Inject
   private Logger logger;
+
+  @Any
+  @Inject
+  private Instance<LogProvider> logProviders;
+  
+  public static final String COLLECTION_NAME = "loginDetails";
```

Comment:
This needs to be something else like userNotifications?

Not sure if the whole mongodb should be rethought (ie user -> logindetails & notificationDetails) though.